src/Microsoft.Azure.SignalR/ServiceRouteBuilder.cs

```diff
@@ -86,14 +86,13 @@ private async Task RedirectToService(HttpContext context, string hubName, IList<
             }
         }
 
-        private void Start<THub>() where THub : Hub
-        {
-            var app = new ConnectionBuilder(_serviceProvider)
-                .UseHub<THub>()
+        private void Start<THub>() where THub : Hub

```

Comment:
I would just remove this method completely and inline it into MapHub since this no longer starts anything.